K8s-Orch-Tools/jira_common.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import sys
+import os
+import time
+
+
+username = os.environ['JIRA_USERNAME']
+password = os.environ['JIRA_PASSWORD']
+
+IMGURL = '/static/img'
+
+
+def userauth():
+    return (username, password)
+
+class FailedJqlException(Exception):
+    def __init__(self, message):
+        '''Helper exception'''
+        super().__init__(message)
+
+
+def query(jql, querydict=None):
+
+    querystring = ''
+    if querydict:
+        querystring = '&'.join(['%s=%s'%(k,v) for k,v in list(querydict.items())]) + '&'
+
+
+    url = "https://jts.seagate.com/rest/api/2/search?%sjql=%s" % (querystring, jql)
+    #url = "https://jts.seagate.com/rest/api/2/search?%sjql=%s&expand=changelog" % (querystring, jql)
+
+    auth = userauth()
+    headers = {
+       "Accept": "application/json",
+       "Content-Type": "application/json",
+    }
+
+    retry_count = 0
+    while True:
+        try:
+            print("%s GET %s" % (time.ctime(), url), file=sys.stderr)
```

Comment:
```suggestion
            print(f"{time.ctime()} GET {url}", file=sys.stderr)
```
`%s` strings are so Python2.